Studio-Lovelies/GG-JointJustice-Unity
# 71 add hold it / objection / take that actions with support for random character specific variations

## Summary

Implements OBJECTION, HOLD_IT, and TAKE_THAT actions.

## Before/after screenshots and/or animated gif

![Kapture 2021-10-19 at 02 17 05](https://user-images.githubusercontent.com/67964179/137828457-1639360d-9f98-4d8d-9e16-2718ba844e45.gif)

## Testing instructions

Run the "ShoutTest" narrative script and check shouts play correctly. Change Arin's random shouts adding audio clips and sprites to the "Shout Variants" array on the Arin scriptable object.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #71
